po/nl.po

```diff
@@ -263,12 +263,12 @@ msgid "%N %V - Article '%T' (%u unread, %t total)"
 msgstr "%N %V - Artikel ‘%T’ (%u ongelezen, %t totaal)"
 
 #: src/configcontainer.cpp:315
-#, fuzzy
 msgid ""
 "%N %V - Search results for '%s' (%u unread, %t total)%?F? matching filter "
 "'%F'&?"
```

Comment:
It looks like [the ampersand in `%?F?...&?` is optional](https://newsboat.org/releases/2.27/docs/newsboat.html#_format_strings):
> The & and [format 2] are optional, i.e. if the identifier’s text is empty, then an empty string will be inserted.

I think the format-string is slightly easier to read without it.
However, I don't think it is worth changing (would generate unnecessary work for all translators)

Reply:
TIL :D I agree that it isn't worth changing though.